docs/intro_tutorial.rst

```diff
@@ -163,6 +163,25 @@ here is the GHZ state :math:`\mathbb{C}^{4^{\otimes 7}}` as
     \frac{1}{\sqrt{30}} \left(| 0000000 \rangle + 2| 1111111 \rangle + 3|
     2222222 \rangle + 4| 3333333\rangle \right).
 
+.. code-block:: python
+
+    >>> from toqito.states import ghz
+    >>> import numpy as np
+    >>> dim = 4
+    >>> num_parties = 7
+    >>> coeffs = [1, 2, 3, 4]
```

Comment:
@Abhishek2634 I think there is a bug in the function. You have to provide the normalized coefficients to get the output close to the latex equation.  

```suggestion
    >>> coeffs = [1/np.sqrt(30), 2/np.sqrt(30), 3/np.sqrt(30), 4/np.sqrt(30)]
```